src/WebJobs.Script.WebHost/Security/Authorization/Policies/AuthorizationOptionsExtensions.cs

```diff
@@ -22,6 +22,12 @@ public static void AddScriptPolicies(this AuthorizationOptions options)
                 p.AddRequirements(new AuthLevelRequirement(AuthorizationLevel.Admin));
             });
 
+            options.AddPolicy(PolicyNames.SystemAuthLevel, p =>
+            {
+                p.AddScriptAuthenticationSchemes();
```

Comment:
Is "AddScriptAuthenticationSchemes" our method, using "Script" as in WebJobs.Script?

Reply:
It is. Just an extension method that add our default/supported schemes to the policy.